gcp/tests/test_qualtrics_scheduling.py

```
@@ -0,0 +1,352 @@
+        assert body["status"] == "success"
+        assert "path" not in body
+        mock_intake_pub.assert_called_once()
+        mock_connect_pub.assert_not_called()
+
+    def test_normal_participant_missing_phone_returns_400(self, normal_payload):
+        """Missing phone on normal (non-Connect) participant returns 400."""
+        normal_payload["PHONE"] = ""
+        with patch.object(
+            _fn1_module, "insert_survey_response", return_value=True
+        ):
+            with _make_request(normal_payload):
```

Ruff (0.15.15)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)